src/main/java/nl/hsac/fitnesse/symbols/RandomString.java

```diff
@@ -41,13 +41,56 @@ public RandomString() {
     }
 
     public String toTarget(Translator translator, Symbol symbol) {
+        String param = symbol.getProperty(MIN_MAX_LENGTH, null);
```

Comment:
*shrug*
This was how I found it, figured there must be a reason for that. But you are right, this is the only place where it is used so why not make it nice :-)